pkg/testutils/mock_flavor.go

```diff
@@ -9,26 +9,24 @@ import (
 )
 
 type MockFlavorMethods struct {
-	FindFlavorFunc func(vm *v1.VirtualMachine) (*flavorv1alpha1.VirtualMachineFlavorProfile, error)
-	ApplyToVmiFunc func(field *k8sfield.Path, profile *flavorv1alpha1.VirtualMachineFlavorProfile, vmiSpec *v1.VirtualMachineInstanceSpec) flavor.Conflicts
+	FindFlavorSpecFunc func(vm *v1.VirtualMachine) (*flavorv1alpha1.VirtualMachineFlavorSpec, error)
+	ApplyToVmiFunc     func(field *k8sfield.Path, flavorspec *flavorv1alpha1.VirtualMachineFlavorSpec, vmiSpec *v1.VirtualMachineInstanceSpec) flavor.Conflicts
 }
 
-var _ flavor.Methods = &MockFlavorMethods{}
```

Comment:
This is a golang pattern to make the compiler verify that the struct implements the interface.
It is useful when changing methods in the interface, so we don't forget to change the struct that implements it.
The variable is probably optimized out on compilation.

Reply:
ACK.